sunpy/io/special/asdf/entry_points.py

```diff
@@ -39,7 +39,7 @@ def get_extensions():
     sunpy_converters = [GenericMapConverter()] + SUNPY_FRAME_CONVERTERS
 
     return [
-        ManifestExtension.from_uri("asdf://sunpy.org/sunpy/manifests/sunpy-1.0.0",
+        ManifestExtension.from_uri("asdf://sunpy.org/sunpy/manifests/sunpy-1.1.0",
```

Comment:
You likely want to add another `ManifestExtension` here (one for `1.0.0` and one for `1.1.0`). The order is also import (newest first so `1.1.0` then `1.0.0`) as asdf will search registered extensions in order.

As this PR is currently written (where `1.0.0` is not installed) a user that opens a file that was created with the `1.0.0` extension will encounter a warning:
```
AsdfWarning: File '...' was created with extension URI 'asdf://sunpy.org/extensions/sunpy-1.0.0' (from package sunpy==5.1.0), which is not currently installed
```

Let me know if more details are helpful. It is fine to register the same converters for both as asdf will filter out converters that don't support any tags in the manifest (as will be the case for `SolarMagnetic` etc and the `1.0.0` extension/manifest).

Reply:
Thanks for pointing this out!  Apparently the one old ASDF file that I happened to try is *so* old that it doesn't trigger this warning, but with more testing I see the behavior you are talking about.  I've made the change as I believe you recommended: 65dcf90